superset-frontend/src/dashboard/components/AddSliceCard/index.jsx

```diff
@@ -111,14 +132,31 @@ function AddSliceCard({
   lastModified,
   sliceName,
   style,
+  thumbnailUrl,
   visType,
 }) {
+  const showThumbnails = useMemo(
+    () => isFeatureEnabled(FeatureFlag.THUMBNAILS),
+    [],
+  );
+
```

Comment:
the value of `isFeatureEnabled(FeatureFlag.THUMBNAILS)` will not change when the Superset web application started, so the `useMemo` might be useless.